### PR TITLE
Release 1.1.facilities fixes

### PR DESCRIPTION
**Pull request by josuegbl**
added new param alignmentAsPriors to consider the priors in relion metadata.
added motioncor2 1.0.5

### CODE REVIEW

File: pyworkflow/em/packages/grigoriefflab/grigoriefflab.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
 import os
 from os.path import join, exists
 
+# Program common names
+FREALIGN_APP = 'FREALIGN'
```
Comment (delarosatrevin) [inline pyworkflow/em/packages/grigoriefflab/grigoriefflab.py]: Why not just FREALIGN and CTFFIND4?

**Comment by delarosatrevin** [comment]
@josuegbl I will review this PR, but first I will like to add some missing parameters (related to number of subsets and subset size) to Relion 2D and 3D classification.

File: pyworkflow/em/packages/grigoriefflab/grigoriefflab.py (hunk 1 above)
Comment (pconesa) [reply pyworkflow/em/packages/grigoriefflab/grigoriefflab.py]: Yep, there is already a constant with that name

Comment (delarosatrevin) [reply pyworkflow/em/packages/grigoriefflab/grigoriefflab.py]: With a different value/purpose?

Comment (pconesa) [reply pyworkflow/em/packages/grigoriefflab/grigoriefflab.py]: Yes! I thought the same, but found that other variable and didn't want to "pull the blanket" (;-))

**Comment by delarosatrevin** [comment]
@josuegbl If the copy alignment as priors is used when the input particles has not alignment information (or this is my guess) it is failing in the convertInputStep, I think it should be validated before executing the protocol. If I launch the same protocol from particles with alignment it seems to work. (see image below)

image

**Comment by josuegbl** [comment]
@delarosatrevin you are right! Ill fix it.

**Pull request merged**